aio/aio-proxy/aio_proxy/routes.py

```diff
@@ -57,6 +57,31 @@ async def search_endpoint(request):
         )
 
 
+@routes.get("/near_point")
+async def near_point_endpoint(request):
+    try:
```

Comment:
Great to see the same `try: / except:` logic as in `search` used here

I wonder what is the best way to share it between endpoint

Reply:
I was thinking about it. 
But I haven't figured out a way to do it yet.

Reply:
Not sur if it applies here, but what I understand of how decorators work, it seems an elegant solution : https://medium.com/swlh/handling-exceptions-in-python-a-cleaner-way-using-decorators-fae22aa0abec

Reply:
@XavierJp I tried refactoring the entire function (try and except blocks), which led to me changing the structure of the search functions' parameters but gathering filters and parameters in one dictionary.
I am not sure if that is pushing the refactoring a little too far. But I could always roll back and only refactor exceptions.